lib/rest/Twilio.ts

```diff
@@ -0,0 +1,934 @@
+  _numbersv2: any
+  _chatv1: any
+  _conversationsv1: any
+  _bulkexportsv1: any
+  _insightsv1: any
+  _trusthubv1: any
+  _flexv1: any
+  _videov1: any
+  _frontlinev1: any
+  _trunkingv1: any
+  _notifyv1: any
+  _pricingv2: any
+  _api: any
+  _studiov1: any
+  _supersimv1: any
+  _monitorv1: any
+  _verifyv2: any
+  _eventsv1: any
+  _voicev1: any
+  _ipmessagingv1: any
+  _mediav1: any
+  _faxv1: any
+  _serverlessv1: any
+  _syncv1: any
+  _studiov2: any
+  _chatv2: any
+  _accountsv1: any
+  _chatv3: any
+  _messagingv1: any
+  _lookupsv1: any
+  _autopilotv1: any
+  _taskrouterv1: any
+  _wirelessv1: any
+  _proxyv1: any
+  _ipmessagingv2: any
+  _pricingv1: any
+
+  constructor(username: string, password: string, opts: opts) {
+    super(username, password, opts);
+
+    //Domains
+    this._numbersv2 = undefined
+    this._chatv1 = undefined
+    this._conversationsv1 = undefined
+    this._bulkexportsv1 = undefined
+    this._insightsv1 = undefined
+    this._trusthubv1 = undefined
+    this._flexv1 = undefined
+    this._videov1 = undefined
+    this._frontlinev1 = undefined
+    this._trunkingv1 = undefined
+    this._notifyv1 = undefined
+    this._pricingv2 = undefined
+    this._api = undefined
+    this._studiov1 = undefined
+    this._supersimv1 = undefined
+    this._monitorv1 = undefined
+    this._verifyv2 = undefined
+    this._eventsv1 = undefined
+    this._voicev1 = undefined
+    this._ipmessagingv1 = undefined
+    this._mediav1 = undefined
+    this._faxv1 = undefined
+    this._serverlessv1 = undefined
+    this._syncv1 = undefined
+    this._studiov2 = undefined
+    this._chatv2 = undefined
+    this._accountsv1 = undefined
+    this._chatv3 = undefined
+    this._messagingv1 = undefined
+    this._lookupsv1 = undefined
+    this._autopilotv1 = undefined
+    this._taskrouterv1 = undefined
+    this._wirelessv1 = undefined
+    this._proxyv1 = undefined
+    this._ipmessagingv2 = undefined
+    this._pricingv1 = undefined
+
+    if (!opts.lazyLoading) {
+        this._numbersv2
+        this._chatv1
+        this._conversationsv1
+        this._bulkexportsv1
+        this._insightsv1
+        this._trusthubv1
+        this._flexv1
+        this._videov1
+        this._frontlinev1
+        this._trunkingv1
+        this._notifyv1
+        this._pricingv2
+        this._api
+        this._studiov1
+        this._supersimv1
+        this._monitorv1
+        this._verifyv2
+        this._eventsv1
+        this._voicev1
+        this._ipmessagingv1
+        this._mediav1
+        this._faxv1
+        this._serverlessv1
+        this._syncv1
+        this._studiov2
+        this._chatv2
+        this._accountsv1
+        this._chatv3
+        this._messagingv1
+        this._lookupsv1
+        this._autopilotv1
+        this._taskrouterv1
+        this._wirelessv1
+        this._proxyv1
+        this._ipmessagingv2
+        this._pricingv1
+    }
+
+  }
+}
+
+Object.defineProperty(Twilio.prototype,
+'numbersv2', {
```

Comment:
You should be able to define these from within the class definition itself: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get